configure.ac

```diff
@@ -78,7 +78,8 @@ m4_define([libva_lt_age],
 m4_define([libdrm_version], [2.4])
 
 # Wayland minimum version number
-m4_define([wayland_api_version], [1.0.0])
+# 1.11.0 for wl_proxy_create_wrapper
+m4_define([wayland_api_version], [1.11.0])
```

Comment:
ubuntu xenial, ubuntu trusty, and fedora 24 don't have packages for wayland 1.11.  Thus, it will break compilation in libva-utils when attempting to use --enable-wayland all around and relying on distro wayland packages.  This will force everyone to compile wayland locally... I don't think that's a good idea!